helix-view/src/editor.rs

```diff
@@ -823,7 +823,7 @@ pub struct Editor {
     pub breakpoints: HashMap<PathBuf, Vec<Breakpoint>>,
 
     pub clipboard_provider: Box<dyn ClipboardProvider>,
-
+    pub last_clipboard: [Vec<String>; 2],
```

Comment:
Very minor nit but I think this field should have docs on it: the name doesn't describe the values well (and I think it's hard to come up with a good name for this field without it being 4 or more words long) so I think it will be difficult to tell what this is used for without checking the reference in `commands.rs`.

```suggestion
    /// The last selection contents yanked to the system and primary clipboards.
    /// These values are saved so that pasting from the system clipboard may
    /// retain multiple selections.
    pub last_clipboard: [Vec<String>; 2],
```

Reply:
Sure. I just added the commit.